### PR TITLE
Add MOTD about duplication

### PR DESCRIPTION
**Pull request by reinhardt**
I'm reusing NewsletterSetting here although the new setting has nothing to do with the newsletter. I'm also not happy that we have to pass all the known values to query the settings. I'm working on a proposal to improve that (#203).

syslabcom/scrum#1137

### CODE REVIEW

File: src/osha/oira/client/browser/dashboard_banner.py [hunk 1]
```diff
@@ -1,14 +1,95 @@
...
+                "text": api.portal.translate(
+                    _(
+                        "Keep updated with the latest developments by signing up for "
+                        "our newsletter on your ${target}",
+                        mapping={"target": preferences_link},
+                    )
+                ),
+                "disabled_key": "call-for-action-banner-disabled",
+            },
+        ]
```
[the remark this answers is not in the document]
Comment (reinhardt) [reply src/osha/oira/client/browser/dashboard_banner.py:62]: There are two lines that cycle over the messages like this. One of them is due to the strange situation with the settings. That leaves only one line. I hesitate to restructure the data just for that one line.

[the remark this answers is not in the document]
Comment (ale-rt) [reply src/osha/oira/client/browser/dashboard_banner.py:62]: Well as mentioned it is not just that...

[the remark this answers is not in the document]
Comment (reinhardt) [reply src/osha/oira/client/browser/dashboard_banner.py:62]: You vaguely mentioned it but did not specify anything else.

[the remark this answers is not in the document]
Comment (ale-rt) [reply src/osha/oira/client/browser/dashboard_banner.py:62]: Neverm ind :)

**Pull request merged**